CPP_STYLE_GUIDE.md

```diff
@@ -33,6 +34,24 @@ these rules:
 
 `char* buffer;` instead of `char *buffer;`
 
+## C++ style comments
+
+Use C++ style comments (`//`) for both single line and multi-line comments.
+
+Examples:
+
+```c++
+// a single line comment
```

Comment:
When we are on it - it would be nice to add that the start of a comment should also be upper cased :smile:

Reply:
And a dot at the end!